client/src/elements/molecules/item-list-entry.css

```diff
@@ -1,27 +1,19 @@
 item-list-entry {
-  background: var(--q-color-primary-2);
-
-  padding: calc(var(--q-space-base) / 2);
-  margin-bottom: calc(var(--q-space-base) / 2);
-
   display: flex;
-
-  transition: transform 0.2s ease-in;
```

Comment:
please, can we keep the transition? :-)

Reply:
Added back in https://github.com/nzzdev/Q-editor/pull/133/commits/f653a255ba161bb98e01d0f914ddbf204da4bd41